qapi/machine.json

```diff
@@ -29,7 +29,7 @@
 { 'enum' : 'SysEmuTarget',
   'data' : [ 'aarch64', 'alpha', 'arm', 'avr', 'cris', 'hppa', 'i386', 'lm32',
              'm68k', 'microblaze', 'microblazeel', 'mips', 'mips64',
-             'cheri128', 'cheri128magic', 'cheri256', 'cheri',
+             'mips64cheri128', 'mips64cheri128magic',
```

Comment:
Dropping cheri(256) is deliberate, those were removed in the meson merge

Reply:
Magic can also go since it's basically unsupported. The only reason I didn't remove it in the meson port was to avoid more unrelated changes.

Reply:
Yeah probably, but the config does still exist so I kept it here to be in sync